app/client/components/identity/EmailAndMarketing.tsx

```diff
@@ -0,0 +1,186 @@
+        const user = await UserAPI.memoRead();
+        if (!user.validated) {
+          window.location.assign(IdentityLocations.VERIFY_EMAIL);
+          return;
+        }
+        const consentOptions = await ConsentOptions.getAll();
+        setEmail(user.email);
+        dispatch(options(consentOptions));
+      } catch (e) {
+        dispatch(error());
+      }
+    };
+    makeInitialAPICalls();
+  }, []);
+
+  const newsletters = ConsentOptions.newsletters(state.options);
+  const consents = ConsentOptions.consents(state.options);
+  const loading = newsletters.length === 0 && consents.length === 0;
+
+  const errorRef = React.createRef<HTMLDivElement>();
+
+  useEffect(
+    () => {
+      if (state.error && errorRef.current) {
+        window.scrollTo(0, errorRef.current.offsetTop - 20);
+      }
+    },
+    [state.error]
+  );
+
+  const errorMessage = (
+    <PageContainer>
+      <div
+        ref={errorRef}
+        css={{
+          fontSize: "13px",
+          lineHeight: "18px",
+          backgroundColor: "#ffe1e1",
+          borderBottom: `1px solid ${palette.red.light}`,
+          borderTop: `1px solid ${palette.red.light}`,
+          color: palette.red.medium,
+          marginTop: "6px",
+          padding: "7px 8px"
+        }}
+      >
+        <p css={{ marginBottom: "10px" }}>Sorry, something went wrong!</p>
+        <p css={{ marginBottom: "0" }}>
+          <a
+            css={{
+              color: palette.blue.dark,
+              cursor: "pointer"
+            }}
+            onClick={() => window.location.reload()}
+          >
+            Refresh this page
+          </a>
+        </p>
+      </div>
+    </PageContainer>
+  );
+
+  const content = (
+    <>
+      <PageContainer>
+        <NewsletterSection
+          newsletters={newsletters}
+          clickHandler={toggleSubscription}
+        />
+      </PageContainer>
+      <PageContainer>
+        <MarginWrapper>
+          <Lines n={4} />
+        </MarginWrapper>
+      </PageContainer>
+      <PageContainer>
+        <ConsentSection consents={consents} clickHandler={toggleSubscription} />
+      </PageContainer>
+      <PageContainer>
+        <MarginWrapper>
+          <Lines n={1} />
+        </MarginWrapper>
+      </PageContainer>
+      <PageContainer>
+        <EmailSettingsSection
+          email={email}
+          actionHandler={setRemoveAllEmailConsents}
+          removed={removed}
+        />
+      </PageContainer>
+      <PageContainer>
+        <MarginWrapper>
+          <Lines n={4} />
+        </MarginWrapper>
+      </PageContainer>
+      <PageContainer>
+        <OptOutSection consents={consents} clickHandler={toggleSubscription} />
+      </PageContainer>
+      <PageContainer>
+        <MarginWrapper>
+          <MembershipLinks />
+        </MarginWrapper>
+      </PageContainer>
+    </>
+  );
+
+  const loader = (
+    <PageContainer>
+      <Spinner loadingMessage="Loading your subscripton information..." />
```

Comment:
subscription?

Reply:
The only piece of copy I did myself and it's the one with the spelling error ...

Reply:
we probably shouldn't be using the word _subscription_ at all on the 'Email & marketing tab' - to avoid confusion with reader revenue _subscriptions_

Reply:
@JessL123

Reply:
Yup Tom's right. I think "Loading your email preferences" would be better